src/modules/components/member/memberDataListItem/memberDataListItemStructure/memberDataListItemStructure.tsx

```diff
@@ -51,7 +51,7 @@ export const MemberDataListItemStructure: React.FC<IMemberDataListItemProps> = (
                         avatarSrc={avatarSrc}
                         responsiveSize={{ md: 'md' }}
                     />
-                    {isDelegate && !isCurrentUser && <Tag variant="info" label="Your Delegate" />}
+                    {isDelegate && !isCurrentUser && <Tag variant="info" label="Your delegate" />}
```

Comment:
Ah interesting. It is "Your delegate" on the Votes design so I went to look at Members and it was lowercase delegate as well so I updated here to match as I know it's a more international convention to use 'Initial case' instead of 'Title Case' (more American.)